e2e-tests/functions

```diff
@@ -744,6 +744,10 @@ spinup_pxc() {
 		apply_config "$config"
 	fi
 
+	# todo(remove after 1.10 release)
+	sleep 10
+	kpatch_set_field svc "${cluster}-pxc-unready" "/spec/publishNotReadyAddresses" true
```

Comment:
@defbin we have discussed it. You need to add  condition for it. Because we can run the tests on different version of k8s.